src/Slider.tsx

```diff
@@ -57,6 +57,7 @@ export const Slider = forwardRef<SliderTypes.API, PropsWithChildren<Settings>>((
     const [isScrollable, setIsScrollable] = useState<boolean>(false);
     const [isDragging, setIsDragging] = useState<boolean>(false);
     const [isBlockingClicks, setIsBlockingClicks] = useState<boolean>(false);
+    const [showNavigationButtons, setShowNavigationButtons] = useState<boolean>(!hideNavigationButtons);
```

Comment:
This starts as the value of !hideNavigationsButtons, but is replaced when setting the controls visibility. I think this breaks the current functionality to not show navigation buttons after sliding.

Reply:
As discussed, the value won't be overwritten because the function for setting the button visibility is never called if the navigation buttons are hidden